Add guides for i18n (#4789)
rotki/rotki: docs/usage_guide.rst

General Settings
----------------------

Language
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Set the language used on the app. This feature is experimental and may not work as expected, with limited supported language, since we need to manually translate the static sentences on the app. Therefore, we need your contribution to speed up the process. You can contribute to the translation `here <https://rotki.readthedocs.io/en/latest/contribute.html#add-new-language-or-translation>`__.

Reviewer: ```suggestion
Set the language used on the app. This feature is experimental and may not work as expected for some languages. This is because all static sentences in the app have to be manually translated and that is a long and ongoing process. Therefore, we need your contribution to speed it up. You can contribute to the translation `here <https://rotki.readthedocs.io/en/latest/contribute.html#add-new-language-or-translation>`__.
```
